reference/common/models/alertsconfig.v1.yaml

```diff
@@ -11,6 +11,7 @@ properties:
     allOf:
       - $ref: './alerts/base.v1.yaml'
       - $ref: './alerts/delay.v1.yaml'
+      - $ref: './alerts/repeat.v1.yaml'
```

Comment:
So the `repeat` only applies to `low` and `high`? Previously it was part of `base` which made it available for all of them. Is that the intention?

Reply:
Yep. I had previously misunderstood the intent.